# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/filters/filter-drop-shadow-clip.png
# Commit: e24ed89246a4
# Full Hash: e24ed89246a4a6c6b4a567ec412f0e3295b4570d
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-04-26 09:46:09
# Regressor Bug: 1766017
# File Overlap Count: 1
# Description:
#   Bug 1766017 - Do a better job of clipping off-screen surface allocation sizes r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D144534
# ==============================================================================

diff -r cdc42b7822f1 -r e24ed89246a4 gfx/wr/wrench/reftests/filters/filter-drop-shadow-clip.png
Binary file gfx/wr/wrench/reftests/filters/filter-drop-shadow-clip.png has changed